src/modules/wallet/components/TokenCard/TokenCard.js

```diff
@@ -26,15 +27,13 @@ const TokenCard = ({ lockedBalance, address, token }) => {
             value={convertFromBaseDenom(availableBalance, token)}
           />
         )}
-        {!lockedBalance ? null : (
-          <DialogLink
-            data-testid="locked-balance"
-            component="lockedBalance"
-            data={{ address }}
+        {lockedBalance > 0 && (
```

Comment:
lockedBalance can be string? so its best to do `BigInt(lockedBalance) > BigInt(0)` for safer check?